ddln_sim_node.proto

```diff
@@ -1,13 +1,14 @@
 syntax = "proto2";
 
-package ddln;
+package ddln.sim_node;
 
 import "ddln_common.proto";
 
 message GroundTruth {
     optional int64 timestamp_ns = 1;
-    optional Pose pose_ENU = 2;
-    optional LatLongAlt lla = 3;
+    optional Pose pose = 2; // Pose in ENU or ECEF coordinate frame
+    enum Type {ENU = 0; ECEF = 1;}
```

Comment:
we don't need Type sent with every pose update - it's not going to change during flight

instead we need an initialization message that will set the origin georeference (e.g. ENU lat/lon/alt, or ECEF x/y/z). Iditally this georeference should be forwarded from `GetWorldInfo` Imaginator call, but not necessarily in this CL.

From VXS sim (imaginator_rec) we call GetWorldInfo, the response should contain georeference.
Then pass the georeference of the start location to the SimNode to initialize its GPS or other global sensor.

We don't have to introduce this logic in this change, but then let's not add `Type` here either.

Reply:
The pose is ambiguous and should provide a type to make sense. We don't know all use cases. It  could as well be a single call.

Reply:
Again, this about providing a get ground truth method, not about initialization. That is a separate issue / task.